### PR TITLE
Multus Tests: Get the image to use for the network config daemon from the virt operator pod

### PR DESCRIPTION
**Pull request by fedepaol**
**What this PR does / why we need it**:

Right now the network setup is done via a daemon that uses the same image as the virt operator (hardcoded from the local registry). 
This won't work in case of external providers, where the local registry may not exist.

In this PR we fetch the image to use directly from the running pod.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note

NONE

```

### CODE REVIEW

File: tests/vmi_multus_test.go [hunk 1]
```diff
@@ -819,6 +819,13 @@ func pingVirtualMachine(vmi *v1.VirtualMachineInstance, ipAddr, prompt string) {
 // both iptables and newer nftables.
 // TODO: Once kubernetes-nmstate is ready, we should use it instead
 func configureNodeNetwork(virtClient kubecli.KubevirtClient) {
+
+	pods, err := virtClient.CoreV1().Pods(tests.KubeVirtInstallNamespace).List(metav1.ListOptions{LabelSelector: "kubevirt.io=virt-operator"})
```
Comment (phoracek) [inline tests/vmi_multus_test.go]: Please add a comment explaining why we do this.

**Comment by booxter** [comment]
/retest

Failed tests are not the ones affected by this change.

**Pull request merged**